wp-content/themes/core/components/statistic/Statistic_Controller.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Tribe\Project\Templates\Components\statistic;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+
+/**
+ * Class Statistic_Controller
+ */
+class Statistic_Controller extends Abstract_Controller {
+	public const TAG     = 'tag';
+	public const CLASSES = 'classes';
+	public const ATTRS   = 'attrs';
+	public const VALUE   = 'value';
+	public const LABEL   = 'label';
+
+	public string $tag;
+	private array $classes;
+	private array $attrs;
+	private $value;
+	private $label;
```

Comment:
@msawicki I got statistics to render (yay) but I had to remove the array type here, I think they should be Deferred_Component? Tried a few ways of doing that and decided PHP is not my first language.